Lombiq.NodeJs.Extensions/scripts/markdown-to-msbuild.js

```diff
@@ -1,37 +1,30 @@
-﻿const findRecursively = require('./find-recursively');
 const fs = require('fs');
-const fsPromises = require('node:fs/promises');
 const markdownlint = require('markdownlint');
 const path = require('path');
 const process = require('process');
-const textlint = require("textlint");
-
-const markdownlintConfig = {
-    default: true,
-    MD013: false,
-    MD033: {
-        allowed_elements: [
-            // A special element in GitHub to indicate a keyboard key. Other Markdown formatters that don't support it will
-            // safely ignore the tags and render the content as inline text without adverse effects.
-            'kbd',
-        ]
-    }
-};
+const textlint = require('textlint');
```

Comment:
Can we do something with this too? 
![image](https://user-images.githubusercontent.com/1976647/180670330-762fc23c-cc73-4f05-8588-04f86ca62d87.png)

Reply:
It should go away if you do a `pnpm install` in the _Lombiq.NodeJs.Extensions_ directory.

Reply:
Why would I do that manually though? Shouldn't this happen by Node.js Extensions?

Reply:
I think I executed it from the wrong working directory. Should be good now.

Reply:
b544650bda1f5394f579e9aaade9bcb3d2d1173d OSOCE commit? There I see the same (after a git clean->reopen->build).

Reply:
I still see this.

Reply:
I don't. I couldn't repro it with a git clean, I even did a fresh clone. I assumed it was accidentally fixed, but maybe it's something on your end? And of course GHA works just as expected: throws warnings when there are problems, doesn't when there aren't. Could you try doing a fresh clone in a new directory?

Reply:
I see this after the mentioned git clean -> VS reopen -> build. Not surprisingly, the same happens in a fresh clone too.

Reply:
Actually, sometimes ignored files don't get cleared out. (Also consider whatever demonic curse is going on in OSOE-110, where the error can be reproduced with a fresh clone but not with git clean nor with dotnet clean.)

Then like I said, please send some logs because I still can't reproduce it.

Reply:
Do you mean a build log? Because these warnings appear in editing time, simply by opening the file.

Reply:
Oh... ... ... Then I wish you started with that or at least didn't crop your screenshot so tightly.

In this case the answer is that I don't care. ;) The cause is that the npm packages aren't installed in the project directory (_Lombiq.NodeJs.Extensions/_), same reason why we don't need _Lombiq.NodeJs.Extensions/pnpm-lock.yaml_ as mentioned [here](https://github.com/Lombiq/NodeJs-Extensions/pull/5#discussion_r930414786). I guess you'd see the same if you open _Lombiq.NodeJs.Extensions/scripts/minify.js_ too, which I haven't touched. The curse of custom build pipelines is that it doesn't work with default tooling. Just have to put up with it.

Reply:
Minify.js actually doesn't show the same issue, nor any of the other scripts in the folder. So, there's something special to just this one.

Reply:
Strange, because `klaw` in minify.js isn't a built-in module either. For some reason the ESLint warnings don't show up for me in VS2022's intellisense at all, so I couldn't verify myself.

Anyway, It's a false positive, so I'm just going to disable them.

Reply:
If you look at [minify.js](https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/scripts/minify.js#L10), you'll find an `eslint-disable` comment right above the `require` call to `klaw` 🤓

So there's nothing mystical going on here.